### PR TITLE
Document Lambda Layer ARNs

### PR DESCRIPTION
**Pull request by cneijenhuis**
#### Summary

Document the Lambda Layer ARNs

#### Description

@lauraluiz @alex-ct and @nikossvnk packaged the SDK into an AWS Lambda Layer and published it.

As each region can contain a different version, let's document each region individually.

#### Todo

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [x] Documentation

### CODE REVIEW

File: lambdaLayerArns.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+AWS Region | Lambda Layer ARN
+--- | ---
+ap-south-1 | arn:aws:lambda:ap-south-1:564387806506:layer:commercetools-nodejs-sdk:9
```
[the remark this answers is not in the document]
Comment (cneijenhuis) [reply lambdaLayerArns.md]: That is the version number of the layer in **this** region.

Unfortunately, the region numbers can deviate from each other. Even if we start from scratch, if a new region is added, it'll start at 1 there again.

The `lambdaLayersArns.md` is generated by the deploy script - yes, the next step is how to integrate that into your release process.

File: README.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ https://commercetools.github.io/nodejs/
 [sdk-middleware-user-agent-dependencies]: https://david-dm.org/commercetools/nodejs?path=packages/sdk-middleware-user-agent
 [sdk-middleware-user-agent-dependencies-icon]: https://img.shields.io/david/commercetools/nodejs.svg?path=packages/sdk-middleware-user-agent&style=flat-square
 
+### SDK as an AWS Lambda Layer
+
+We also a publish an [AWS Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-using) with the packages above. Find the ARN for your region [here](lambdaLayerArns.md).
```
Comment (nikossvnk) [inline README.md]: To be precise, not all packages above are included. Package `@commercetools/sdk-auth` was left out. Also package `api-request-builder` from category `Other` is included

Comment (cneijenhuis) [reply README.md]: Ops, thanks! Is that better: https://github.com/commercetools/nodejs/pull/1118/commits/1496933ad7a16d76390beb4f2a0a37509a949ccb ?

File: lambdaLayerArns.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (daern91) [reply lambdaLayerArns.md]: Sounds good, maybe we could find some time this week on how to integrate the script :- )

File: README.md (hunk 2 above)
Comment (nikossvnk) [reply README.md]: nice :)

**Pull request merged**